packages/block-library/src/image/view.js

```diff
@@ -355,6 +438,14 @@ const { state, actions, callbacks } = store(
 				}
 			`;
 			},
+			setScreenReaderText() {
+				const { ref } = getElement();
```

Comment:
It is now possible to use `@wordpress/a11y` module and `speak` method. See the dev note from @sirreal: https://make.wordpress.org/core/2024/10/14/updates-to-script-modules-in-6-7/.

Example usage in `@wordpress/interactivity-router` for reference:

https://github.com/WordPress/gutenberg/blob/b7d989e0dafb23866f65295a851e6b8734c3b693/packages/interactivity-router/src/index.ts#L421-L425